src/connectors/connect-aws/connect-s3.ts

```diff
@@ -59,14 +59,16 @@ export class ConnectS3 extends Connection {
       throw new Error("failed to upload data " + JSON.parse(result.body).message);
     }
     this.parents = [event.cid];
-    return null;
+    return undefined
```

Comment:
why the change? I have been wrapping my head around null vs undefined with no concrete answers. This is a public signature, in the sense that all connector authors must deal with it.